docs/content/get-started/playground.md

```diff
@@ -67,7 +67,7 @@ with Aperture. There is an instance of Grafana running on the cluster as well
 for viewing metrics from experiments.
 
 The Playground is preloaded with a
-[Latency Gradient Concurrency Control](/tutorials/flow-control/dynamic-rate-limiting.md)
+[Latency Gradient Concurrency Control](/tutorials/flow-control/concurrency-limiter.md)
```

Comment:
It is `basic-concurrency-limiting.md`

Reply:
Just saw the error in the pipeline, and found the issue; anyway, thanks for pointing it out.